sechub-api-java/src/main/java/com/mercedesbenz/sechub/api/SecHubClient.java

```diff
@@ -129,4 +129,7 @@ public interface SecHubClient {
      */
     void approveJob(String projectId, UUID jobUUID) throws SecHubClientException;
 
-}
\ No newline at end of file
+    String getServerVersion() throws SecHubClientException;
+
+    void userRequestsNewApiToken(String emailAddress) throws SecHubClientException;
```

Comment:
This is not an eventhandler where methods are in style like "what happened" but more what the caller WANTS to do.

Her my suggestion:
```suggestion
    void requestNewApiToken(String emailAddress) throws SecHubClientException;
```